packages/phone-number-privacy/combiner/package.json

```diff
@@ -1,27 +1,21 @@
 {
   "name": "@celo/phone-number-privacy-combiner",
-  "version": "3.2.1",
```

Comment:
We shouldn't be manually changing version numbers in PRs. 

add a changeset an mark it as a minor update

Reply:
Followed up on Slack, changes were already added to CHANGELOGS. The changeset was removed automatically when the common package was published